apps/src/code-studio/components/progress/BulkLessonVisibilityToggle.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import {connect} from 'react-redux';
+import {getStore} from '../../../redux';
+import PropTypes from 'prop-types';
+import Button from '@cdo/apps/templates/Button';
+import FontAwesome from '../../../templates/FontAwesome';
+import ReactTooltip from 'react-tooltip';
+import {toggleHiddenLesson} from '@cdo/apps/code-studio/hiddenLessonRedux';
+import {unitCalendarLesson} from '../../../templates/progress/unitCalendarLessonShapes';
+import _ from 'lodash';
+import style from './bulk-lesson-visibility-toggle.module.scss';
```

Comment:
these are entirely unordered, and some use an absolute syntax while others use a relative syntax. are there any standards for this?

Reply:
We don't have any standards for order unfortunately :( 

For absolute vs relative syntax: I'll use relative if it's in the the files is in the same directory and otherwise use `@cdo/apps`. I'm not sure if that's just a personal preference though